doc/examples/reconst_dki.py

```diff
@@ -161,26 +167,11 @@
 """
 Note that these four standard measures could also be computed from Dipy's DTI
 module. For comparison purposes, we calculate below the FA, MD, AD, and RD
-using Dipy's DTI module. For this, we first extract the data of the lower
-b-value to decrease the effects of the signal components lager than the 2nd
-order which are not taken into account by DTI.
+using Dipy's TensorModel.
```

Comment:
Worth saying that the values of these parameters are different in DKI and DTI (and why)